java/client/src/main/java/glide/api/commands/ServerManagementClusterCommands.java

```diff
@@ -316,6 +318,72 @@ public interface ServerManagementClusterCommands {
      */
     CompletableFuture<ClusterValue<Long>> lastsave(Route route);
 
+    /**
+     * Deletes all the keys of all the existing databases. This command never fails.<br>
+     * The command will be routed to all primary nodes.<br>
+     * To explicitly specify the flushing mode, use {@link #flushall(FlushMode)}.
+     *
+     * @see <a href="https://redis.io/commands/flushall/">redis.io</a> for details.
+     * @return Aggregated response - <code>OK</code>.
+     * @example
+     *     <pre>{@code
+     * String response = client.flushall().get();
+     * assert response.equals("OK");
+     * }</pre>
+     */
+    CompletableFuture<String> flushall();
+
+    /**
+     * Deletes all the keys of all the existing databases. This command never fails.<br>
+     * The command will be routed to all primary nodes.
+     *
+     * @see <a href="https://redis.io/commands/flushall/">redis.io</a> for details.
+     * @param mode The flushing mode, could be either {@link FlushMode#SYNC} or {@link
+     *     FlushMode#ASYNC}.
+     * @return Aggregated response - <code>OK</code>.
```

Comment:
Are you ok with such doc? `redis-rs` aggregates the responses.
(but it is always `OK`)

Reply:
I don't think the "Aggregated response" is necessary.  I think anyone who knows anything will understand that a command that "never fails" will return an OK.